docker-compose.yml

```diff
@@ -30,3 +30,4 @@ services:
       - .env
     volumes:
       - .:/app
+      - /app/node_modules
```

Comment:
@jstvz what's this for?

Reply:
Because we're using a volume that mounts the local project in the web
image. The volume isn't mounted during the build, so node_modules wasn't showing up, which meant yarn couldn't find the commands to run `yarn serve`.

Reply:
Ah, I get it now: this overlays /app/node_modules from the image on top of /app mounted from outside the container. We also need this for `sfdx`; I'll add it.